app/config/locale/translations/hi.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "hi",
     "settings.direction": "ltr",
     "emails.sender": "%s टीम",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "खाता सत्यापन",
+    "emails.verification.hello": "नमस्ते {{name}}",
+    "emails.verification.body": "अपना ईमेल सत्यापित करने के लिए इस लिंक का अनुसरण करें।",
+    "emails.verification.footer": "यदि आपने इस पते को सत्यापित करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.verification.thanks": "धन्यवाद",
+    "emails.verification.signature": "{{project}} टीम",
+    "emails.magicSession.subject": "लॉग इन",
+    "emails.magicSession.hello": "नमस्ते,",
+    "emails.magicSession.body": "लॉग इन करने के लिए इस लिंक का पालन करें।",
+    "emails.magicSession.footer": "यदि आपने इस ईमेल का उपयोग करके लॉगिन करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.magicSession.thanks": "धन्यवाद",
+    "emails.magicSession.signature": "{{project}} टीम",
+    "emails.recovery.subject": "पासवर्ड रीसेट",
+    "emails.recovery.hello": "नमस्ते {{name}}",
+    "emails.recovery.body": "अपना {{project}} पासवर्ड रीसेट करने के लिए इस लिंक का अनुसरण करें।",
+    "emails.recovery.footer": "यदि आपने अपना पासवर्ड रीसेट करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.recovery.thanks": "धन्यवाद",
+    "emails.recovery.signature": "{{project}} टीम",
+    "emails.invitation.subject": "%s टीम को %s पर आमंत्रण",
+    "emails.invitation.hello": "नमस्ते",
+    "emails.invitation.body": "यह मेल आपको इसलिए भेजा गया था क्योंकि {{owner}} आपको {{project}} पर {{team}} टीम का सदस्य बनने के लिए आमंत्रित करना चाहता था।",
+    "emails.invitation.footer": "यदि आप रुचि नहीं रखते हैं, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.invitation.thanks": "धन्यवाद",
+    "emails.invitation.signature": "{{project}} टीम",
     "locale.country.unknown": "अज्ञात",
```

Comment:
A lot of these translations have been picked up from Google Translate and seem to lack a natural flow. Please fix them

Reply:
We do not prefer translations from Google Translate

Reply:
Can I fix this?

Reply:
I have tried to be more original this time. Please have a look.